wiki/Gameplay/Comboburst/de.md

```diff
@@ -0,0 +1,19 @@
+---
+stub: true
+tags:
+  - milestone
+  - comboburst
+  - Meilenstein
+---
+
+# Combo-Bilder
+
+**Combo-Bilder** (auch *Combo-Bursts* genannt) lobpreisen die Erreichung eines [Combo-Meilensteins](/wiki/Gameplay/Combo_milestone) durch das kurzzeitige Aufblitzen eines Bildes. Diese Bilder sind in jedem Spielmodus unterschiedlich.
+
+- In [osu!](/wiki/Game_mode/osu!) und [osu!catch](/wiki/Game_mode/osu!catch) erscheinen Charaktere entweder von der linken oder rechten Seite des Fensters. Standardmäßig sind die erwähnten Charaktere [pippi](/wiki/Mascots#pippi) bzw. [Yuzu](/wiki/Mascots#yuzu).
+- In [osu!taiko](/wiki/Game_mode/osu!taiko) erscheint hinter [pippidon](/wiki/Mascots#don) eine Ansammlung von Blumen.
+- In [osu!mania](/wiki/Game_mode/osu!mania) erscheint standardmäßig ein Charakter an der rechten Seite der Bühne.
+
+In osu!, osu!taiko und osu!catch schießen für Combos, die kleiner als 100 sind, Sterne entweder aus der linken oder rechten Seite des Spielfeldes. Für Combos größer 100 schießen die Sterne aus beiden Seiten gleichzeitig heraus. In osu!mania schießen die Sterne aus der linken und rechten Seite der Bühne heraus.
```

Comment:
```suggestion
In osu!, osu!taiko und osu!catch schießen für Combos, die kleiner als 100 sind, Sterne entweder aus der linken oder rechten Seite des Spielfeldes. Für Combos über 100 schießen die Sterne aus beiden Seiten gleichzeitig heraus. In osu!mania schießen die Sterne aus der linken und rechten Seite der Bühne heraus.
```
Feels wrong to me, but I might just not have seen it before. Can't find any info either way.

Reply:
Sounds better 👍